### PR TITLE
Magnum skins

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
More gamba for the gamba box.

image

Removes some old references to f12 for scopes
Fixed a bayonet item sprite that wasn't updated when the mob sprites were.
## Why It's Good For The Game
Gambling is good. Marines didn't need those points anyway.
## Changelog
:cl: Sprites: Ocelot
add: R-76 magnum is now available in gold, silver and nickle flavours via black market lootboxes in req
/:cl:

### CODE REVIEW

**Comment by zee11zee** [comment]
LETS GOOOO!!!

**Comment by maikilangiolo** [comment]
Cool, now make the gun actually good instead of a reskinned slug shotgun with windup and less ammo

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -723,6 +729,12 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	icon_state = "mosinscope"
 	desc = "A Mosin specific mounted zoom sight scope. Allows zoom by activating the attachment. Use F12 if your HUD doesn't come back."
 
+/obj/item/attachable/scope/standard_magnum
+	name = "R-76 rail scope"
+	desc = "A custom rail mounted zoom sight scope designed specifically for the R-76 Magnum. Allows zoom by activating the attachment. Use F12 if your HUD doesn't come back."
```
Comment (ivanmixo) [inline code/modules/projectiles/gun_attachables.dm:734]:
```suggestion
	desc = "A custom rail mounted zoom sight scope designed specifically for the R-76 Magnum. Allows zoom by activating the attachment."
```

Uh let's not continue this trend. There isn't a case where using a scope should ever make your hud disappear.

**Review by ivanmixo — CHANGES_REQUESTED** [review]

**Comment by Lumipharon** [comment]
dunno why the icon diff bot is losing its mind.

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**